internal/pkg/diff/diff.go

```diff
@@ -165,10 +164,6 @@ func (d *Differ) newOptions(reporter *Reporter) cmp.Options {
 		cmp.Reporter(reporter),
 		// Compare Config/ConfigRow configuration content ("orderedmap" type) as map (keys order doesn't matter)
 		cmp.Transformer("orderedmap", utils.OrderedMapToMap),
-		// Compare strings by line by line
-		cmpopts.AcyclicTransformer("strByLine", func(s string) []string {
-			return strings.Split(s, "\n")
```

Comment:
Drobnost: 
- povodne sme `strings` v `diff` porovnavali tak, ze sa porovnavala kazda line osobitne
- toto riesenie je problematicke, kedze staci, ze jedna `line` je zmazana, ... a potom sa vsetky nasledujuce oznacia ako zmenene
- v pouzitej kniznici `cmp` sa to neda jednoducho vyriesit, `strings` porovna spravne, ale nevieme vypisat pekny diff